core/src/main/scala/org/apache/spark/deploy/JsonProtocol.scala

```diff
@@ -76,12 +76,13 @@ private[deploy] object JsonProtocol {
   }
 
   def writeMasterState(obj: MasterStateResponse): JObject = {
+    val alive_workers = obj.workers.filter(_.isAlive())
```

Comment:
Hi Andrew 

WorkerState is org.apache.spark.deploy.master's private object. 
JsonProtocol cannot resolved WorkerState due to class visibility.